pkg/imgpkg/bundle/bundle_images_lock.go

```diff
@@ -137,7 +137,7 @@ func (o *Bundle) buildAllImagesLock(throttleReq *util.Throttle, processedImgs *p
 	return bundles, processedImageRefs, nil
 }
 
-func (o *Bundle) fetchImagesRef(img regv1.Image, ui util.UIWithLevels) (ImageRefs, error) {
+func (o *Bundle) fetchImagesRef(img regv1.Image, ui util.UIWithLevels, offline bool) (ImageRefs, error) {
```

Comment:
im ok with this for now in favor of getting the build out, but we should probably discuss a more elegant approach.